Convert my sql

youngabreu1 opened the pull request.

Elielvitor45 committed:
Chore: Converção querys PostGreSQL para MySQL

Elielvitor45 committed:
Update README.md

Elielvitor45 committed:
fix: Corrigindo conexão

Elielvitor45 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

Elielvitor45 committed:
docs: Adicionando Configurações

youngabreu1 committed:
style: Implementado disable no botao ao salvar

Ao clicar em Salvar no modal adicionar contato é necessário que o botão seja initilizado enquanto a requisição não seja realizada

youngabreu1 committed:
fix: fixed DashTicketsAndTimes query

youngabreu1 committed:
fix: fixed DashTicketsChannels query

youngabreu1 committed:
fix: fixed DashTicketsEvolutionByPeriod query

youngabreu1 committed:
fix: fixed DashTicketsEvolutionChannels query

youngabreu1 committed:
fix: fixed DashTicketsPerUsersDetail query

youngabreu1 committed:
fix: fixed DashTicketsQueue query

youngabreu1 committed:
fix: fixed StatisticsPerUsers query

youngabreu1 committed:
docs: package-lock

Elielvitor45 committed:
Fix: Convertendo a syntax das Querys DashTickets

Elielvitor45 committed:
fix: Convertendo as querys do ListCampaignService

youngabreu1 committed:
fix: alterando querys em postgres para mysql

Elielvitor45 committed:
fix: Alterando Syntax da Tipagem TypeScript

Elielvitor45 committed:
Fix: Correção das queryes DashTickets

Elielvitor45 committed:
fix: Modificando o operador @ para : das querys

Basicamente as variaveis das querys vao ao banco com os valores direto, entao nao deve colocar @ para demonstrar que e uma variavel do MySql, e sim : para o sequelizer reconhecer como variavel.

Elielvitor45 committed:
fix: Criando um metodo que corrige a geracao de qrcode

O metodo basicamente limpa o campo qrcode do banco de dados, quando e solicitado um novo qrcode

youngabreu1 committed:
fix: correção do método apagarPastaSessao

Elielvitor45 committed:
fix: alterando o Update

Alterei o update, pois a função principal não estava chamando a secundaria, então a solução encontrada, foi fazer o update dentro da função principal

youngabreu1 committed:
fix: conversão de query pgsql para mysql

Elielvitor45 committed:
fix: Modificando a collate para o mesmo do Banco de Dados MySql

Elielvitor45 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

Elielvitor45 committed:
fix: Corrigindo query MySql

Elielvitor45 committed:
fix: Modificando temporariamente a validação

Retirando temporaria a validação 'CheckValidContact' e adicionando um if pois a função CheckValidContact não está tendo um funcionamento correto, e precisa ser analisada para descobrir onde exatamente está o erro dentro dela.

Elielvitor45 committed:
fix: Correções Visando alguns erros

Esses correções foram visando arrumar erros de sql.

Elielvitor45 committed:
fix: Removendo correção Anterior

Removendo correção desnecessaria no Update dos contatos

Elielvitor45 committed:
fix: Corrigindo ChatBot inativo(incompleto)

Corrigindo o envio de mensagem do chatbot quando inativo, atualmente quando inativo, ele envia apenas uma mensagem vazia, o que também deve ser resolvido.

Elielvitor45 committed:
fix: Limitando para 1 fila temporariamente

youngabreu1 committed:
fix: type error destiny.add()

youngabreu1 committed:
fix: main menu options

foi alterado o domínio utilizado para que apareçam as opções API e Campanhas

youngabreu1 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

# Conflicts:
#	backend/src/services/ChatFlowServices/VerifyStepsChatFlowTicket.ts

Elielvitor45 committed:
fix: Atualização em tempo real resovida

Anteriormente tinha uma modificação no SocketEmit por conta de um erro, mas essa modificação estava atrapalhando a atualização em tempo real, após retirar essa modificação, a atualização em tempo real está funcionando corretamente.

youngabreu1 committed:
style: mudança na logo

Elielvitor45 committed:
fix: adicionando uma conexao a um novo banco de dados

youngabreu1 committed:
fix: conexão com banco asterisk

Elielvitor45 committed:
fix: Adicionando service CreateMessageCloseService

Primeiro: Corrigi bug que o ChatBot funciona mesmo quando a loja estiver fechada.

Segundo: Limitei a mensagem de fechamento para ser respondida 3 vezes apenas e não virar spam

Elielvitor45 committed:
fix: Corrigindo o envio de string vazia pelo chatbot

Elielvitor45 committed:
feat: Limite de tempo sem resposta

Adicionar um limite de tempo para tickets sem resposta.
Observacao: Apenas tickets que estao sendo respondidos pelo chatbot se enquadrao para essa limite.

youngabreu1 committed:
fix: Criação de filas com o mesmo nome

não é mais possível realizar a criação de uma fila já existente

Elielvitor45 committed:
feat: Criando o fluxo de validação do Pas no banco do Asterisk

youngabreu1 committed:
fix: linha comentada que indica pro usuário erro

youngabreu1 committed:
feat: implementando a opção de PAS no frontEnd

Elielvitor45 committed:
Merge branch 'BackEndAsterisk' into ConvertMySql

youngabreu1 committed:
fix: loading infinito nos contatos

youngabreu1 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

Elielvitor45 committed:
fix: Correcoes no BackEnd em relacao ao PAS

Elielvitor45 committed:
fix: commit

Elielvitor45 committed:
fix: Algumas correcoes no funcionamento do chatbot

youngabreu1 committed:
feat: implementando o envio de mensagem para fila

O cliente recebe uma mensagem indicando que está sendo redirecionado a uma fila

youngabreu1 committed:
feat: implementando transferir ticket para fila

Elielvitor45 committed:
fix: Adicionando transferencia de fila

youngabreu1 committed:
chore: ignorando arquivo

youngabreu1 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

Elielvitor45 reviewed (changes requested):
Moço, não pode retirar o arquivo asteriskDB.ts é a mesma coisa que retirar o database.ts moço, eu falei ontem para não commitar isso.

Elielvitor45 committed:
fix: Corrigindo commit anterior que retirou arquivo que conecta com o banco de dados do asterisk

youngabreu1 committed:
fix: opção qualquer resposta nas condições

Elielvitor45 committed:
fix: correção gitignore

Elielvitor45 committed:
feat: implementando um fluxo de delete logico para o chatflow

Elielvitor45 committed:
fix: Algumas alterações

Elielvitor45 committed:
fix: Corrigindo mensagem ao iniciar o atendimento

Elielvitor45 committed:
feat: Adicionando limite de mensagens paralelas

Adicionei um limite para mensagens paralelas de um mesmo ticket, antes se o usuario enviasse 10 mensagens de uma vez, o chatbot responderia 10 vezes, agora ira responder apenas uma vez.

Elielvitor45 committed:
fix: Corrigindo transferencia para um Usuario

Ao transferir para um usuario anteriormente, não era transferido para a fila que ele estava atrelado, atualmente com essa alteração, caso você transfira para um usuario atrelado a uma fila, vai dar update no ticket mudando o queueId e o UserId ao mesmo tempo.

Elielvitor45 committed:
feat: Adicionando limite de validacao do PAS e bloqueando numero de contato ao acessar o setor de suporte

Elielvitor45 committed:
feat: Adicionando um campo de descrição nos cantos e removendo os campos anteriores de informaçoes  adicionais

youngabreu1 committed:
fix: contatos agora aceitam apenas 8 digitos

youngabreu1 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

Elielvitor45 committed:
Merge branch 'ConvertMySql' into RemoveAdditionalInformation

youngabreu1 committed:
feat: associação de PAS ao ticket

Ao seguir o fluxo de suporte com o chat bot, caso seja informado um PAS válido, esse pas é associado ao ticket

Elielvitor45 committed:
fix: Adicionando validacao nos Relatorios

Elielvitor45 committed:
Merge branch 'RemoveAdditionalInformation' into ConvertMySql

Elielvitor45 committed:
feat: Adicionando relatorio por PAS

Elielvitor45 committed:
feat: Adicinando insert na tabela Atendimentos em tickets associados a um PAS

Elielvitor45 committed:
fix: mudando o estilo dos cards no menu DashBoard

youngabreu1 committed:
feat: implementando últimas ligações do asterisk

youngabreu1 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

Elielvitor45 committed:
fix: Terminado a correcao do menu DashBoard

youngabreu1 committed:
feat: implementando CheckLastCallService

youngabreu1 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

Elielvitor45 committed:
fix: Melhorando a validação da edição e criação dos contatos

youngabreu1 committed:
fix: corrigindo mensagem de erro na validação nome

Elielvitor45 committed:
fix: Algumas validações extras no frontend

youngabreu1 committed:
feat: Implementando associação manual de PAS

Agora o atendente pode inserir o código PAS manualmente

youngabreu1 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

Elielvitor45 committed:
fix: Possibilitando a criação de Ticket quando o Usuario conectado manda mensagem ao cliente

Elielvitor45 committed:
fix: Corrigindo a exclusão das Mensagens agendadas

Elielvitor45 committed:
fix: Corrigindo a validação de mensagens enviadas via grupo

Elielvitor45 committed:
fix: Corrigindo bug de ao mandar mensagem com o chat desativado, ele ainda responder as tentativas falhas

Elielvitor45 committed:
fix: Configurando o arquivo nginx para rodar em dominio

Elielvitor45 committed:
feat: Configurando arquivo .gitignore

youngabreu1 committed:
fix: Modal associar pas

youngabreu1 committed:
Merge branch 'ConvertMySql' of https://github.com/Elielvitor45/izing.open.io---mySQL into ConvertMySql

Elielvitor45 committed:
feat: Atualizando Backend_izing
